apps/api/src/app/content-templates/usecases/compile-template/compile-template.spec.ts

```diff
@@ -61,6 +62,34 @@ describe('Compile Template', function () {
       })
     );
 
+    it('should allow the user to specify handlebars helpers'), async function () {
+      const result = await useCase.execute(
+        CompileTemplateCommand.create({
+          templateId: 'custom',
+          data: {
+            branding: {
+              color: '#e7e7e7e9',
+            },
+            message: 'hello world'
+          },
+          customTemplate: '<div>{{titlecase message}}</div>'
+        })
+      )
+    };
+
+    Handlebars.registerHelper('titlecase', function(string) {
+      string.trim().toLowerCase().replace(/\w\S*/g, (w) => (w.replace(/^\w/,
+      (c) => c.toUpperCase())));
+    });
+
+    Handlebars.registerHelper('allLower', function(string) {
```

Comment:
```suggestion
    Handlebars.registerHelper('lowercase', function(string) {
```

Reply:
So actually to put this helpers here will only make them work inside the test file :) We need to move the handle bars definition itself to the compile-message usecase for it to work in other cases :)